apix-integrationtests/build.gradle

```diff
@@ -59,11 +59,9 @@ subprojects {
                 'Bundle-ClassPath': bundleClassPath(configurations.compile),
                 'Bundle-SymbolicName': "${project.group}.${project.name}",
                 'Bundle-Description': 'Alfred API: Integration tests',
-                'Import-Package': 'net.sf.cglib.core,'
```

Comment:
Same issue here, should we make this change specific to 6.2?

Reply:
Hm.. I don't think so, since all tests work and they all use the same version of DE which (I think) solves it.